monks-little-details.js

```diff
@@ -703,44 +703,96 @@ background-color: rgba(0, 0, 0, 0.5);
         var apl = { count: 0, levels: 0 };
         var xp = 0;
 
-        //get the APL of friendly combatants
-        for (let combatant of combat.combatants) {
-            if (combatant.actor != undefined && combatant.token != undefined) {
-                if (combatant.token.disposition == 1) {
-                    apl.count = apl.count + 1;
-                    let levels = 0;
-                    if (combatant.actor.system?.classes) {
-                        levels = Object.values(combatant.actor.system?.classes).reduce((a, b) => {
-                            return a + (b?.levels || b?.level || 0);
-                        }, 0);
-                    } else {
-                        levels = combatant?.actor.system.details?.level?.value || combatant?.actor.system.details?.level || 0;
-                    }
-
-                    apl.levels += levels;
-                } else {
-                    let combatantxp = combatant?.actor.system.details?.xp?.value;
-                    if (combatantxp == undefined) {
+        if (game.system.id == 'pf2e') {
+            //cr will just be a -1 to 3 value (representing trivial - extreme)
+            //apl will not be passed forward, instead XP is passed forward
+
+            //note, should be referenced by xpByRelLevel[relLevel + 4]
+            var xpByRelLevel = [10, 15, 20, 30, 40, 60, 80, 120, 160];
+
+            //note that this needs to be multiplied by party size
+            var crsByXp = [10, 15, 20, 30, 40]
+            
+            //determine APL, and modifiers if necessary
+            for (let combatant of combat.combatants) {
+                if (combatant.actor != undefined && combatant.token != undefined) {
+                    if (combatant.token.disposition == 1) {

```

Comment:
Since you've split this into two code blocks, this extra if statement is extraneous and can be added to the one preceding it.

Reply:
Agreed, added to this statement